### PR TITLE
pkg add: fcitx5-chinese-addons

### PR DESCRIPTION
**Pull request by eatradish**
https://gitlab.com/fcitx/fcitx5-chinese-addons

### CODE REVIEW

File: extra-i18n/fcitx5-chinese-addons/autobuild/defines
Comment (liushuyu) [inline extra-i18n/fcitx5-chinese-addons/autobuild/defines]:
```suggestion
PKGDES="Fcitx 5 Chinese addons"
```

File: extra-i18n/fcitx5-chinese-addons/autobuild/defines
Comment (liushuyu) [inline extra-i18n/fcitx5-chinese-addons/autobuild/defines]: I don't see why it needs cURL. Explain your reason.

File: extra-i18n/fcitx5-chinese-addons/autobuild/defines
Comment (liushuyu) [inline extra-i18n/fcitx5-chinese-addons/autobuild/defines]:
```suggestion
BUILDDEP="extra-cmake-modules boost git fmt"
```

File: extra-i18n/fcitx5-chinese-addons/autobuild/defines
Comment (liushuyu) [inline extra-i18n/fcitx5-chinese-addons/autobuild/defines]:
```suggestion
PKGDEP="curl libime qt-5 fcitx5-qt boost opencc fcitx5 gettext"
```

**Review by liushuyu — CHANGES_REQUESTED** [review]
Address this slew of issues.

File: extra-i18n/fcitx5-chinese-addons/autobuild/defines
Comment (liushuyu) [inline extra-i18n/fcitx5-chinese-addons/autobuild/defines]:
```suggestion
BUILDDEP="extra-cmake-modules git fmt"
```

Remove the `BUILDDEP` that is already part of `PKGDEP`

**Review by liushuyu — CHANGES_REQUESTED** [review]
Remove redundant BUILDDEP

File: extra-i18n/fcitx5-chinese-addons/autobuild/defines
Comment (eatradish) [reply extra-i18n/fcitx5-chinese-addons/autobuild/defines]: my fault.....I see PKGBUILD, but i have no tested...

Comment (liushuyu) [reply extra-i18n/fcitx5-chinese-addons/autobuild/defines]: Okay, looks like the cloud Pinyin plugin needs `cURL`

**Comment by liushuyu** [comment]
Syntax-wise looks good, please do a test build.

File: extra-i18n/fcitx5-chinese-addons/autobuild/defines
Comment (liushuyu) [inline extra-i18n/fcitx5-chinese-addons/autobuild/defines]:
```suggestion
PKGDEP="curl libime fcitx5-qt boost opencc gettext"
```

**Review by liushuyu — CHANGES_REQUESTED** [review]
Remove redundant `PKGDEP`

File: extra-i18n/fcitx5-chinese-addons/autobuild/defines
Comment (eatradish) [reply extra-i18n/fcitx5-chinese-addons/autobuild/defines]: why remove fcitx5?

Comment (eatradish) [reply extra-i18n/fcitx5-chinese-addons/autobuild/defines]: and curl can be remove?

Comment (eatradish) [reply extra-i18n/fcitx5-chinese-addons/autobuild/defines]: ahh...ok, cURL looks like the cloud Pinyin plugin needs

Comment (liushuyu) [reply extra-i18n/fcitx5-chinese-addons/autobuild/defines]: >why remove fcitx5?

You already included `fcitx5-qt` which implies the inclusion of `qt-5` and `fcitx5`.

Comment (eatradish) [reply extra-i18n/fcitx5-chinese-addons/autobuild/defines]: > > why remove fcitx5?
> 
> You already included `fcitx5-qt` which implies the inclusion of `qt-5` and `fcitx5`.

ok

**Comment by eatradish** [comment]
> Syntax-wise looks good, please do a test build.

build successfully, you can consider pushing to explosive

**Comment by liushuyu** [comment]
>build successfully, you can consider pushing to explosive

Blocked by #1499.
I will do the build test and merge this after clearing the blocking PR.

**Review by liushuyu — APPROVED** [review]

**Pull request merged**
